maps/site53/site53-1.dmm

```diff
@@ -4057,7 +4075,7 @@
 /obj/effect/catwalk_plated,
 /obj/machinery/camera/network/lcz{
 	name = "SCP-500";
-	network = list("Heavy Containment Zone Network")
+	network = list("Heavy                Containment                Zone                Network")
```

Comment:
You shouldn't be randomly adding spaces, and camera networks are decided by the camera type not entered manually. For HCZ just use the HCZ preset for the cameras.

Reply:
im not adding random spaces its some shit that strongDMM does

it looks fine ingame

Reply:
also i didnt edit cameras in the first place